docs/distributed/README.md

```diff
@@ -40,13 +40,14 @@ Install Minio - [Minio Quickstart Guide](https://docs.minio.io/docs/minio-quicks
 
 To start a distributed Minio instance, you just need to pass drive locations as parameters to the minio server command. Then, you’ll need to run the same command on all the participating nodes.
 
-*Note* 
+*Note*
 
 - All the nodes running distributed Minio need to have same access key and secret key for the nodes to connect. To achieve this, you need to export access key and secret key as environment variables on all the nodes before executing Minio server command.
+- All the nodes running distributed Minio need to be on homogenous environments i.e same network interconnect, same number of drives, same number of disks.
```

Comment:
drives and disks feels repetitive. How about `same operating system, same network interconnect, same number of drives/disks.`

Reply:
should we mention about same number and model or size of disks? I think drive and disk can be used interchangeably. So, maybe disks alone is good enough.

Reply:
done